.nycrc

```diff
@@ -0,0 +1,9 @@
+{
+  "reporter": [
+    "lcov"
+  ],
+  "extension": [
+    ".jsx"
```

Comment:
Do we know if this will prevent it from creating coverage reports for regular `.js` files?